catalog/task/create-github-release/0.1/create-github-release.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: create-github-release
+spec:
+  description: >-
+    Tekton task that creates a release on github.com via the GitHub API
+  params:
+    - name: repository
+      type: string
+      description: "The github repository where the release should be created"
+    - name: release_version
+      type: string
+      description: "The version string of the new release"
+    - name: githubSecret
+      type: string
+      description: "The kube secret to use to authenticate to GitHub, containing one key: token"
+    - name: content_directory
+      type: string
+      description: "The directory inside the workspace to find files for release"
+      default: "release/"
```

Comment:
FYI, we currently use a shared PVC / workspace for all PipelineRuns (unless a PVC is specified in the ReleaseStrategy CR). (It seems we will be moving to volume templates in the future.) So to avoid runs from interfering with each other, we always use a "random" subdirectory inside the workspace, so this default will not really be used. It's fine to have it. But just something to keep in mind - the Pipeline that calls this Task will definitely need to define this.

Reply:
Then, let's remove the default.

Reply:
Updated in 0e89bfa.